beacon-chain/flags/config.go

```diff
@@ -45,8 +46,15 @@ func ConfigureGlobalFlags(ctx *cli.Context) {
 	if ctx.GlobalBool(ArchiveAttestationsFlag.Name) {
 		cfg.EnableArchivedAttestations = true
 	}
-	if ctx.GlobalBool(MinimumHandshake.Name) {
-		cfg.MinimumHandshakes = MinimumHandshake.Value
-	}
+	configureMinimumPeers(ctx, cfg)
+
 	Init(cfg)
 }
+
+func configureMinimumPeers(ctx *cli.Context, cfg *GlobalFlags) {
+	cfg.MinimumSyncPeers = ctx.GlobalInt(MinSyncPeers.Name)
+	maxPeers := int(ctx.GlobalInt64(cmd.P2PMaxPeers.Name))
+	if cfg.MinimumSyncPeers > maxPeers {
+		cfg.MinimumSyncPeers = maxPeers
```

Comment:
Should this log out that we've changed their configuration value?  Could cause confusion otherwise.

Reply:
alright,added it in